modules/messages/client/components/QuickReply.js

```diff
@@ -44,11 +44,36 @@ export default function QuickReply({ onSend, onFocus }) {
           if (write) {
             onFocus();
           } else {
-            onSend(`
+            if (host) {
+              let result = '';
+              const characters =
+                'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+              const charactersLength = characters.length;
+              for (let i = 0; i < 15; i++) {
+                result += characters.charAt(
+                  Math.floor(Math.random() * charactersLength),
+                );
+              }
+              alert(result);
+              const jitsi_random = result;
```

Comment:
This risk intersecting with an existing jitsi room. I bet there's a way to just query jitsi for a random room and use that.